numpy/lib/tests/test_nanfunctions.py

```diff
@@ -686,7 +688,7 @@ def test_extended_axis_invalid(self):
 
     def test_float_special(self):
         with warnings.catch_warnings(record=True):
-            warnings.simplefilter('ignore', RuntimeWarning)
+            warnings.simplefilter('always', RuntimeWarning)
```

Comment:
OK. The record was never looked at anyway, so we could probably do without a filter of anysort. However, using `supress_warnings` might be more informative as that it what is going on.